src/Integrations/ActiveCampaign/SettingsActiveCampaign.php

```diff
@@ -278,6 +278,13 @@ function ($option) use ($formId) {
 				$beforeContent = \apply_filters($filterName, '') ?? '';
 			}
 
+			$sortingButton = Components::render('sorting');
+
+			$formViewDetailsIsEditableFilterName = Filters::getIntegrationFilterName(self::SETTINGS_TYPE_KEY, 'fieldsSettingsIsEditable');
+			if (\has_filter($formViewDetailsIsEditableFilterName)) {
+				$sortingButton = \__('This integration sorting and editing is disabled because of the active filter in your project!', 'eightshift-forms');
```

Comment:
I'm not sure how this is rendered on the front, but maybe this could be styled to notify the user that the integration sorting and ediging has been disabled (like this being italic, or bold). Just a suggestion.

Reply:
On the settings all the fields are disabled so the user can see that there is not editing ability this is just like additional  note for them